spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Fusion.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.mkldnn
+
+import com.intel.analytics.bigdl.Module
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, Activity}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.Node
+
+private[mkldnn] object Fusion {
+
+  private val fuse = System.getProperty("bigdl.mkldnn.fusion", "false").toBoolean
```

Comment:
I assume fusion normally always benefit inference,  in what case we may not want to do fusion?

Reply:
Maybe we can set it default to true after intergating with int8